rosa_sts_unmanaged_oidc/variables.tf

```diff
@@ -0,0 +1,283 @@
+  type        = number
+}
+
+
+variable "autoscaling_enabled" {
+  description = "Enables autoscaling. This variable requires you to set a maximum and minimum replicas range using the `max_replicas` and `min_replicas` variables."
+  type        = string
+  default     = "false"
+}
+
+variable "min_replicas" {
+  description = "The minimum number of replicas for autoscaling."
+  type        = number
+  default     = null
+}
+
+variable "max_replicas" {
+  description = "The maximum number of replicas not exceeded by the autoscaling functionality."
+  type        = number
+  default     = null
+}
+
+variable "proxy" {
+  default     = null
+  description = "cluster-wide HTTP or HTTPS proxy settings"
+  type = object({
+    http_proxy              = string           # required  http proxy
+    https_proxy             = string           # required  https proxy
+    additional_trust_bundle = optional(string) # a string contains contains a PEM-encoded X.509 certificate bundle that will be added to the nodes' trusted certificate store.
+    no_proxy                = optional(string) # no proxy
+  })
+}
+
+#Private link cluster Info
+variable "enable_private_link" {
+  type        = bool
+  description = "This enables private link"
+  default     = false
+}
+
+#VPC Info
+variable "vpc_name" {
+  type        = string
+  description = "VPC Name"
+  default     = "mobb-tf-vpc"
+}
+
+variable "vpc_cidr_block" {
+  type        = string
+  description = "value of the CIDR block to use for the VPC"
+  default     = "10.0.0.0/16"
+}
+
+variable "private_subnet_cidrs" {
+  type        = list(any)
+  description = "The CIDR blocks to use for the private subnets"
+  default     = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+}
+
+variable "public_subnet_cidrs" {
+  type        = list(any)
+  description = "The CIDR blocks to use for the public subnets"
+  default     = ["10.0.101.0/24", "10.0.102.0/24", "10.0.103.0/24"]
+}
+
+variable "aws_subnet_ids" {
+  type = list(any)
+  description = "A list of AWS Subnet IDs if you have created the VPC separately"
+  default = null
+}
+
+variable "single_nat_gateway" {
+  type        = bool
+  description = "Single NAT or per NAT for subnet"
+  default     = false
+}
+
+#AWS Info
+variable "aws_region" {
+  type    = string
+  default = "us-east-2"
+}
+
+variable "availability_zones" {
+  type        = list(any)
+  description = "The availability zones to use for the cluster"
+  default     = ["us-east-2a", "us-east-2b", "us-east-2c"]
+}
+
+#Azure AD config
+variable "aad_app_name" {
+  type        = string
+  default     = "kumudu-tf-test-app"
+  description = "Azure AD App Name"
+}
+
+variable "aad_app_password_name" {
+  type        = string
+  default     = "kh-ROSA-Secret"
+  description = "AAD Secret Name"
+}
+
+variable "aad_app_redirect_uri" {
+  type        = string
+  default     = "https://oauth-openshift.apps.iv2wbc7f.westus2.aroapp.io/oauth2callback/AAD"
```

Comment:
We should remove this default.

Reply:
The whole thing isn't used right now, but indeed